src/nerve/core/mcp/__init__.py

```
@@ -0,0 +1,26 @@
+    >>> await client.close()
+"""
+
+from nerve.core.mcp.client import MCPClient, MCPToolInfo
+from nerve.core.mcp.errors import MCPConnectionError, MCPError
+
+__all__ = [
+    "MCPClient",
+    "MCPToolInfo",
+    "MCPError",
+    "MCPConnectionError",
+]
```

Ruff (0.14.10)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)